tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+envlist = black, flake8
+skipsdist = true
+
+[testenv]
+basepython = python3.6
+changeenv = {toxinidir}
+
+[base]
+changedir =
+scriptfiles =
+    test/run-tests
+
+[testenv:black]
+deps = black
+
+commands = black --py36 --check --diff \
+    {[base]scriptfiles}
+
+[testenv:flake8]
+deps = flake8
+commands =
+    flake8 --statistics {posargs} \
+    {[base]scriptfiles}
+
+[flake8]
+show_source = True
+max-line-length = 88
+
+[travis]
+python = 3.6: black,flake8
```

Comment:
ERROR:   py37: Error creating virtualenv. ... Error details: InvocationError('Failed to get version_info for python3.6: b"pyenv: python3.6: command not found"
Could it be related to 3.6 here vs.  3.7-dev in the first hunk?

Reply:
I removed 3.7-dev from travis for now since we do not have any unit tests that would make sense to run against different python versions.